pkg/crc/machine/start.go

```diff
@@ -417,6 +418,16 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	}
 	logging.Info("CRC VM is running")
 
+	if err := lockCoreUserPasswd(sshRunner); err != nil {
+		return nil, errors.Wrap(err, "Error deleting the password for core user")
```

Comment:
Should this be moved to an `else` in `if startConfig.EmergencyLogin {` as both operations exclusive